### ISSUE TITLE
Error messages not associated with form fields

### ISSUE THREAD

**Issue by RyanFH257** [localgovdrupal/localgov#830]
Whilst working on a microsite form, I noticed that the error messages were not programatically associated with their form fields. This means the error messages are not read out by screen readers.

I checked our main council website and saw it was an issue there too. I also checked a few other council sites and found the same issue.

This seems to be an issue with the webform module and not an LGD specific one, but it would be good if we could fix it and pass it back.

How it currently is

![Image](https://github.com/user-attachments/assets/dff89b76-20db-4498-8a41-402ef06034f3)

How it should be

![Image](https://github.com/user-attachments/assets/77a86d53-3867-44f6-9899-cce5285fb3b8)

**Comment by msayoung** [comment]
I think this issue might be relevant: https://www.drupal.org/project/drupal/issues/3083103

**Comment by msayoung** [comment]
https://git.drupalcode.org/project/localgov_forms/-/work_items/3584161
